README.md

```diff
@@ -1 +1,76 @@
-# Enumerable_ruby
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Enumerable
+
+![screenshot]()
+
```

Comment:
- [ ] Kindly add a description of this project and kindly remove the `screenshot` since it has no path or you can add a link to your screenshot if you want to have your screenshot in your Readme. This is to make your Readme more informative and professional.

Reply:
I fixed it @julie-ify. Thanks 👍🏼